templates/flower/flower-deployment.yaml

```
@@ -48,6 +48,19 @@ spec:
         - name: {{ template "registry_secret" . }}
       {{- end }}
       containers:
+        {{- if (and .Values.authSidecar.enabled ( eq .Values.executor "CeleryExecutor" )) }}
```

```suggestion
        {{- if .Values.authSidecar.enabled }}
```
This whole doc is already only shown for CeleryExecutor so you dont need to duplicate the check here.

---

nice catch @pgvishnuram ^

---

changes applied